_rules/role-attribute-valid-value-674b10.md

```diff
@@ -30,26 +30,31 @@ acknowledgments:
 
 ## Applicability
 
-Any `role` attribute that is neither empty ("") nor only [ASCII whitespace][], and that is specified on an HTML or SVG element that is [included in the accessibility tree][].
+Any [`role` attribute][role attribute] for which all the following are true:
```

Comment:
Linking attributes is only necessary if they are ambiguous. There is only one type of role attribute in HTML.
```suggestion
Any `role` attribute for which all the following are true:
```